source/backend/support/task.cpp

```diff
@@ -103,7 +103,7 @@ POV_LONG Task::ConsumedCPUTime() const
 void Task::Start(const boost::function0<void>& completion)
 {
     if((done == false) && (taskThread == NULL))
-        taskThread = NewBoostThread(boost::bind(&Task::TaskThread, this, completion), 1024 * 1024 * 2); // TODO - make stack size definable
+        taskThread = NewBoostThread(boost::bind(&Task::TaskThread, this, completion), POV_THREAD_STACK_SIZE);
```

Comment:
What I found was boost doesn't have a standard stack size on linux & likely any platform. It inherits whatever the parent process stack setting is and this exposes us to the stack size being too small (or just different to our own) due a particular user's environment. 

Today anyone post Jan 16' and >Boost 1.5 gets a 2MB hard coded stack. I believe we should stick with this hard coded minimum values approach on all platforms given the opaque fail signature and that we know existing user scenes need this amount or more.

"POV_THREAD_STACK_SIZE" therefore needs to be defined for all platforms. The macro itself apparently in part due Doxygen. If we use it directly, want to adjust for 64 bits on as many platforms as possible, adjust again for linux and still be able to detect whether the user has set his own value so we don't override it - things get somewhat complicated... As for following the form for the headers files, I suppose I think I reasonably did. :-)  

Anyway, I'm not going to burn a lot more time here. Give me a bit to see if I can get something simpler working which tactically fixes the known regression fail on linux and still lets us set POV_THREAD_STACK_SIZE. If I can, I'll force an update here later this morning, otherwise I'll close this req.